### PR TITLE
First PR Req from  Testing

### PR DESCRIPTION
**Pull request by sshelake25**

### CODE REVIEW

**Review by ShivamChawla1199 — COMMENTED** [review]
Comment

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>React App - Contact list agge</title>
```
Comment (maniroop16) [inline public/index.html]: This is Testing Comment

Comment (Shaurya018) [reply public/index.html]: comment 1

Comment (sshelake25) [reply public/index.html]: updated as per comment added on index file

**Review by Shaurya018 — APPROVED** [review]

**Review by ShivamChawla1199 — APPROVED** [review]

**Review by maniroop16 — APPROVED** [review]

**Pull request merged**